src/main/java/org/example/LessonDao.java

```
@@ -5,51 +5,46 @@
-
-            statement.executeUpdate();
-        } catch (SQLException e) {
-            throw new SQLException("Error adding homework: " + e.getMessage());
-        }
-    }
-
-    public void addLesson(Lesson lesson) throws SQLException {
+    public void addLesson(Lesson lesson) {
         String query = "INSERT INTO Lesson (name, homework_id) VALUES (?, ?)";
+        int nameIndex = 1;
+        int homeworkIdIndex = 2;
 
         try (Connection connection = DataBaseConnection.getConnection();
              PreparedStatement statement = connection.prepareStatement(query)) {
 
-            statement.setString(1, lesson.getName());
-            statement.setInt(2, lesson.getHomework().getId());
+            statement.setString(nameIndex, lesson.getName());
+
+            if (lesson.getHomework() != null) {
+                statement.setInt(homeworkIdIndex, lesson.getHomework().getId());
+            } else {
+                statement.setNull(homeworkIdIndex, java.sql.Types.INTEGER);
+            }
 
             statement.executeUpdate();
         } catch (SQLException e) {
-            throw new SQLException("Error adding lesson: " + e.getMessage());
+            System.err.println("Error adding lesson: " + e.getMessage());
         }
     }
 
-    public void deleteLesson(int lessonId) throws SQLException {
+
+    public void deleteLesson(int lessonId) {
         String query = "DELETE FROM Lesson WHERE id = ?";
+        int idIndex = 1;
 
         try (Connection connection = DataBaseConnection.getConnection();
              PreparedStatement statement = connection.prepareStatement(query)) {
 
-            statement.setInt(1, lessonId);
+            statement.setInt(idIndex, lessonId);
 
             statement.executeUpdate();
         } catch (SQLException e) {
-            throw new SQLException("Error deleting lesson: " + e.getMessage());
+            System.err.println("Error deleting lesson: " + e.getMessage());
```

throw an exception instead of using a print statement